src/css/main.less

```diff
@@ -838,36 +838,25 @@ fieldset[disabled] .btn-info.active {
   background-color: #5e646b;
 }
 
-.health-dot {
-  display: inline-block;
-}
 
-.health-dot:before {
-  content: "\00a0";
-  border-radius: 50%;
-  border: solid 2px #51565d;
-  width: 12px;
-  height: 12px;
-  display: inline-block;
-  vertical-align: bottom;
-  margin-right: 6px;
-  margin-bottom: 1px;
-}
+/* ==============
```

Comment:
Could you remove this comment?
The class names are already self-explanatory and we don't have comments on other places in this file, so this would be inconsistent.